### ISSUE TITLE
Operator KerasLayer not support

### ISSUE THREAD

**Issue by onexuan** [MarsTechHAN/keras2ncnn#26]
[ERROR] Operator KerasLayer not support.
=========================================
{'name': 'keras_layer', 'trainable': True, 'dtype': 'float32', 'handle': 'https://tfhub.dev/google/imagenet/mobilenet_v2_140_224/feature_vector/4', 'batch_input_shape': [None, 224, 224, 3]}

**Comment by onexuan** [comment]
[ERROR] Operator BatchNormalizationV1 not support.
=========================================
{'name': 'bn_Conv1', 'gamma_constraint': None, 'center': True, 'gamma_regularizer': None, 'gamma_initializer': {'class_name': 'Ones', 'config': {'dtype': 'float32'}}, 'moving_variance_initializer': {'class_name': 'Ones', 'config': {'dtype': 'float32'}}, 'beta_constraint': None, 'momentum': 0.999, 'moving_mean_initializer': {'class_name': 'Zeros', 'config': {'dtype': 'float32'}}, 'dtype': 'float32', 'axis': [3], 'beta_initializer': {'class_name': 'Zeros', 'config': {'dtype': 'float32'}}, 'beta_regularizer': None, 'epsilon': 0.001, 'scale': True, 'trainable': True}

**Comment by MarsTechHAN** [comment]
Hi, can you provide your h5df file here? I only find the savedmodel format in the tfhub.

**Comment by onexuan** [comment]
https://github.com/GantMan/nsfw_model/releases/download/1.2.0/mobilenet_v2_140_224.1.zip

**Comment by MarsTechHAN** [comment]
See it, let me have a try

**Comment by MarsTechHAN** [comment]
hmmm... I dont see a clear way of supporting the KerasLayer. It does not contain any structural information in the model, just the weight.
![image](https://user-images.githubusercontent.com/10462805/109613337-f65ded80-7b6b-11eb-8334-21b2bca9a436.png)

**Comment by onexuan** [comment]
I got
